engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -300,4 +295,15 @@ public int compare(RenderableChunk o1, RenderableChunk o2) {
         }
     }
 
+	@Override
+	public RenderingConfig getRenderingConfig() {
```

Comment:
I assume it won't need the worldProvider or the renderingConfig, since the method that needs them is only called for the playerCamera in worldRenderer(which is never an OrtographicCamera) so I'll leave it as is (but if it does need the worldProvider and renderingConfig, it'd be better to inject them, since ShadowMapNode.java doesn't instance the worldProvider)